.circleci/config.yml

```diff
@@ -136,6 +153,29 @@ workflows:
           requires:
             - request-test-approval
 
+      - request-demo-approval:
+          type: approval
+          requires:
+            - helm_lint
+            - build_docker_for_demo
+          filters:
+            branches:
+              only:
+                - /demo-.*/
```

Comment:
Wonder if this is entirely necessary? I imagine one single approval step would be enough to trigger a build _and_ the deploy